docs/source/intervention_models/maternal_malaria_prevention_therapy/index.rst

```diff
@@ -0,0 +1,200 @@
+
+#. Sleep under insecticide-treated bednets
+#. Are treated for malaria illness and anaemia
+#. Receive chemoprevention with an effective antimalarial drug during the second and third trimesters 
+
+This intervention model document details the third portion of this WHO recommendation to prevent malaria in pregnant women living in malraia endemic areas.
+
+.. list-table:: Abbreviations
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+    - Note
+  * - IPTp
+    - intermittent preventive treatment (for malaria) in pregnancy 
+    - 
+  * - IPTp-SP
+    - intermittent preventive treatment (for malaria) in pregnancy with sulfadoxine-pyrimethamine
+    - 
+  * - ANC
+    - Antenatal care
+    - 
+
+.. todo::
+
+  Fill out table with any abbreviations and their definitions used in this document.
+
+Intervention Overview
+-----------------------
+
+The WHO recommendation for a positive pregnancy (2016) on intermittent preventive treatment in pregnancy (IPTp) for malaria prevention states the following: "In malaria-endemic areas in Africa, intermittent preventive treatment with sulfadoxine-pyrimethamine (IPTp-SP) is recommended for all pregnant women. Dosing should start in the second trimester, and doses should be given at least one month apart, with the objective of ensuring that at least three doses are received" (page xiv/16)
+
+Additionally the WHO publication Guidelines for the treatment of malaria (2015) states "WHO recommends that, in areas of moderate-to-high malaria transmission of Africa, IPTp-SP be given to all pregnant women at each scheduled ANC visit, starting as early as possible in the second trimester, provided that the doses of SP are given at least 1 month apart. WHO recommends a package of interventions for preventing malaria during pregnancy, which includes promotion and use of insecticide-treated nets, as well as IPTp-SP”. To ensure that pregnant women in endemic areas start IPTp-SP as early as possible in the second trimester, policy-makers should ensure health system contact with women at 13 weeks of gestation." 
+
+This is considered to be a strong recommendation based on high quality evidence.
+
+.. list-table:: Affected Outcomes
+  :widths: 15 15 15 15
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Modeled?
+    - Note
+  * - Maternal anemia
+    - Reported as protective RR
+    - Not yet
+    - [Radeva-Petrova-et-al-2014]_
+  * - Maternal uncomplicated clinical malaria
+    - Reported as protective RR
+    - Not yet
+    - [Radeva-Petrova-et-al-2014]_
+  * - Infant birthweight
+    - Reported as increase in population mean as well as protective RR for low birthweight
+    - Yes
+    - [Radeva-Petrova-et-al-2014]_. Statistically insignificant reduction in preterm birth
```

Comment:
"Statistically insignificant" you say... sounds like this will not be an important part of our story, then.